tensorrt_llm/_torch/auto_deploy/models/hf.py

```diff
@@ -84,9 +84,7 @@ def _get_max_position_embeddings_config(self) -> Dict[str, Any]:
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
-
-        self._quant_config: Optional[Dict] = None
-
+        self._quant_config_reader: Optional[QuantConfigReader] = None
```

Comment:
```suggestion
        self._quant_config_reader: QuantConfigReader | None = None
```
This is the preferred syntax since python 3.10, it in use in many places throughout the repository.